Documentation/PageTranslations/Index.rst

```diff
@@ -6,10 +6,10 @@
 Translating pages
 =================
 
-When you want to translate a page in the page tree you create an
-"alternative page language" record on that page. This contains fields
-similar to that of the page record of the default language which you fill in
-with translated content.
+In order to translate a page in the page tree you create a
+copy of that record starting by its parent page. 
+It will be generated in the default language with fields prefixed by a text similar to "[Translate to Language X:]". 
```

Comment:
```suggestion
It will be generated in the default language with all page property fields (like the title, description and others), prefixed by a text similar to "[Translate to Language X:]". 
```

Reply:
Don't mention "and others" here.

Reply:
adjusted, feel free to apply